### ISSUE TITLE
refresh LoC

### ISSUE THREAD

**Issue by GlenDC** [pajecawav/ghloc-web#79]
Hi,

It seems that once a LoC is established that it doesn't update anymore to a new amount, despite the code changing.
I can confirm that it does seem to be aware of the new commits and stargazers,
butt hat it does not update the actual code that it displays for the repo and also not the LOC.

**Comment by pajecawav** [comment]
Hi, do you have an example of a repo with wrong LoC data displayed on https://ghloc.vercel.app/ ?

We have a cache for each repo on the backend, but I think LoC's are cached for like 20 minutes or so. Maybe you are seeing results from the cache and have to wait a little bit?

**Comment by GlenDC** [comment]
Don't think that is the issue, I have seen same LoC since weeks.

<https://github.com/plabayo/rama>

I went to your website and there I can see the code it displays (source code) is out of date, which I guess explains the outdated LoC count. Other stats on that page are however up to date (e.g. stargazer count).

<https://ghloc.vercel.app/plabayo/rama?branch=main&filter=.rs>

Now it is however updated. Used to display around ~64k. I checked now and the badge (and website) now both show around ~72k. 

I suppose you found the issue as this was definitely unchanged for many weeks prior to this, despite the code base slowly growing all this time.

Either way, thank you for looking into it!

I'll close it for now as I think it's resolved.
